internal/config/supervisor/types.go

```diff
@@ -3,17 +3,46 @@
 
 package supervisor
 
-import "go.pinniped.dev/internal/plog"
+import (
+	"strings"
+
+	"go.pinniped.dev/internal/plog"
+)
 
 // Config contains knobs to setup an instance of the Pinniped Supervisor.
 type Config struct {
 	APIGroupSuffix *string           `json:"apiGroupSuffix,omitempty"`
 	Labels         map[string]string `json:"labels"`
 	NamesConfig    NamesConfigSpec   `json:"names"`
 	LogLevel       plog.LogLevel     `json:"logLevel"`
+
+	SupervisorHTTPListener  string `json:"supervisorHTTPListener"`
+	SupervisorHTTPSListener string `json:"supervisorHTTPSListener"`
```

Comment:
```suggestion
	SupervisorHTTPListener  *string `json:"supervisorHTTPListener"`
	SupervisorHTTPSListener *string `json:"supervisorHTTPSListener"`
```

Unset / `null` in the YAML means "use the default from the Go code"